modules/oe_theme_helper/src/Plugin/PageHeaderMetadata/NodeViewRoutesBase.php

```diff
@@ -51,19 +37,13 @@ abstract class NodeViewRoutesBase extends PageHeaderMetadataPluginBase implement
    *   The plugin_id for the plugin instance.
    * @param mixed $plugin_definition
    *   The plugin implementation definition.
-   * @param \Drupal\Core\Routing\RouteMatchInterface $current_route_match
-   *   The current route match.
-   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
-   *   The entity type manager.
-   * @param \Drupal\Core\Entity\EntityRepositoryInterface $entity_repository
-   *   The entity repository.
+   * @param \Symfony\Component\EventDispatcher\EventDispatcherInterface $event_dispatcher
+   *   The event dispatcher.
    */
-  public function __construct(array $configuration, string $plugin_id, $plugin_definition, RouteMatchInterface $current_route_match, EntityTypeManagerInterface $entity_type_manager, EntityRepositoryInterface $entity_repository) {
+  public function __construct(array $configuration, string $plugin_id, $plugin_definition, EventDispatcherInterface $event_dispatcher) {
```

Comment:
Unfortunately this break backwards compatibility. There might be websites using this plugin as base class. We need to leave in place the already injected services. In scope of this ticket, let's create a PR for 2.x where we remove both the deprecated plugin and the unneeded services.